### PR TITLE
make log more clear so that easy to debug

### PR DESCRIPTION
**Pull request by lowang-bh**
add not found job podgroup's namespace/name in log

### CODE REVIEW

**Comment by lowang-bh** [comment]
@wangyang0616 @hwdef plz help to take a look at this commit. thanks.

**Review by hwdef — COMMENTED** [review]
/lgtm

**Comment by lowang-bh** [comment]
@william-wang Please have a look, thanks.

File: docs/design/Enhance-Generate-PodGroup-OwnerReferences-for-Normal-Pod.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ When controller need create a podgroup, it will check the pod `ownerReferences`
 
 ### DiscoveryClient & DynamicClient
 From the `ownerReferences`, we can get or create a GVK and owner resource name. 
-Discovery client is foucs on the k8s resources, it can get GVR from GVK. When get GVR and owner resource name, we can use Dynamic client to get the owner resource's `ownerReferences`.
+Discovery client is focus on the k8s resources, it can get GVR from GVK. When get GVR and owner resource name, we can use Dynamic client to get the owner resource's `ownerReferences`.
```
Comment (hzxuzhonghu) [inline docs/design/Enhance-Generate-PodGroup-OwnerReferences-for-Normal-Pod.md:71]: focused

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**